### PR TITLE
Petting the honks

### PR DESCRIPTION
**Pull request by xTrainx**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: xTrainx
add: Petting Honkers now makes it honk
/:cl:

[why]: # fixes issue #8281

### CODE REVIEW

**Comment by ghost** [comment]
HONK
+1

**Comment by JohnGinnane** [comment]
thank you for this

File: hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -37,6 +37,15 @@
 	speak = list("Meow!", "Honk!", "Haaaa....", "Hink!")
 	speak_chance = 15
 	emote_see = list("shakes its head.", "shivers.", "does a gag.", "clowns around.")
+	var clowndown = 0
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm]: I think you'll need a "/" between "var" and the name, i.e. "var/clowndown"

File: hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 2]
```diff
@@ -37,6 +37,15 @@
...
+	var clowndown_time = 100
+	
+/mob/living/simple_animal/pet/cat/clown/attack_hand(mob/living/user)
+	..()
+	if(user.a_intent == "help")
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm]: need to check `if (user.a_intent == "help" && clowndown < world.time)` to check if cooldown is off

**Review by JohnGinnane — DISMISSED** [review]
need some fixes

File: hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 3]
```diff
@@ -37,6 +37,15 @@
...
+		visible_message("[name] is so happy it let's out a honk!")
+		playsound(src, pick(meows), 100)
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm]: where is meows defined?

File: hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 4]
```diff
@@ -37,6 +37,15 @@
 	speak = list("Meow!", "Honk!", "Haaaa....", "Hink!")
 	speak_chance = 15
 	emote_see = list("shakes its head.", "shivers.", "does a gag.", "clowns around.")
+	var clowndown = 0
+	var clowndown_time = 100
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm]: tbh 10 seconds is too long, i think 1 is enough

File: hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 5]
```diff
@@ -37,6 +37,18 @@
...
 	speak_chance = 15
 	emote_see = list("shakes its head.", "shivers.", "does a gag.", "clowns around.")
+	var/clowndown = 0
+	var/clowndown_time = 100
+	
+/mob/living/simple_animal/pet/cat/clown/attack_hand(mob/living/user)
+	..()
+	if(user.a_intent == "help")
+		if(clowndown > world.time)
+			visible_message("[name] purss a bit while you pet it...")
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/simple_animal/friendly/cat.dm]: This should be "purs" I think?

**Comment by Jujumatic** [comment]
priority: CRITICAL

**Review by Jujumatic — APPROVED** [review]

**Review by Jujumatic — APPROVED** [review]

**Comment by Jujumatic** [comment]
this is a fix so no head input needed

**Pull request merged**